driver-sync/src/main/com/mongodb/client/vault/ClientEncryptions.java

```diff
@@ -16,26 +16,26 @@
 
 package com.mongodb.client.vault;
 
-import com.mongodb.KeyVaultEncryptionSettings;
-import com.mongodb.client.internal.KeyVaultImpl;
+import com.mongodb.ClientEncryptionSettings;
+import com.mongodb.client.internal.ClientEncryptionImpl;
 
 /**
- * Factory for KeyVault implementations.
+ * Factory for ClientEncryption implementations.
  *
  * @since 3.11
  */
-public final class KeyVaults {
+public final class ClientEncryptions {
```

Comment:
This is a lousy name now.  Not sure what to do with it.

Reply:
ClientEncryptionFactory?

Reply:
What do you think @rozza ?  ClientEncryptions is consistent with our other factories, but it makes no sense.

Reply:
I can live with it.